feature-utils/poly-analysis/src/analysis/report/inactive-cards-summary.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+
+/**
+ * I am a "fake" analysis that shows issues when running analyses.
+ * I do not have an analyse method as I get the data directly in the constructor.
+ *
+ * I provide the same API as an analysis that should be included in a report.
+ *
+ * @class
+ */
+export default class InactiveCardsSummary {
+    constructor(analysesResults) {
+        const inactiveAnalysesResults = analysesResults.filter(
+            ({ analysis, status }) => status.isError || !analysis.active
+        );
+
+        this._inactiveAnalysesData = inactiveAnalysesResults.map(
+            ({ analysis, status }) => {
+                return {
+                    analysis: analysis.id,
+                    activationStatus: analysis.active ? "ACTIVE" : "INACTIVE",
+                    executionStatus: status.name,
+                    message: status.message,
+                };
+            }
+        );
+        this.active = this._inactiveAnalysesData.length > 0;
+    }
+
+    get title() {
+        return "Inactive Analyses";
+    }
+
+    get id() {
+        return this.constructor.name;
+    }
+
+    get jsonReport() {
+        return {
+            id: this.id,
+            data: this._inactiveAnalysesData,
+        };
+    }
+
+    render() {
+        return (
+            <>
+                <p>
+                    Analyses that were not activated. This is a technical view
```

Comment:
Are we ever showing this to the end user? Again, separation of concerns.
Additionally, I think we should separate layers of abstraction here... We are already including visualization in a layer that's called "analysis".

Reply:
will be moved to a separate PR :D